pkg/apis/infrastructure/v1alpha2/awsmachinetemplate_types.go

```diff
@@ -0,0 +1,63 @@
+/*
+Copyright 2019 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package v1alpha2
+
+import (
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	clusterv1 "sigs.k8s.io/cluster-api/pkg/apis/cluster/v1alpha2"
+)
+
+// AWSMachineTemplateSpec defines the desired state of AWSMachineTemplate
+type AWSMachineTemplateSpec struct {
+	Template AWSMachineTemplateResource `json:"template"`
+}
+
+// AWSMachineTemplateResource describes the data needed to create am AWSMachine from a template
+type AWSMachineTemplateResource struct {
+	metav1.TypeMeta `json:",inline"`
+
+	// Standard object's metadata.
+	clusterv1.ObjectMeta `json:"metadata,omitempty"`
```

Comment:
Shouldn't this be metav1.ObjectMeta?

Reply:
Nevermind!

Reply:
Given that this is the Spec's template for AWSMachine, it should use our version of ObjectMeta without the extra validation